### PR TITLE
MavenProject

### PR DESCRIPTION
**Pull request by Jena-Biswajit**

### CODE REVIEW

File: src/BubbleSort.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+public class BubbleSort
+{
+    public static void bubbleSort(int[] arr)
+    {
+        //traverse array
```
Comment (rkpatra201) [inline src/BubbleSort.java:5]: do code formatting. @Jena-Biswajit

File: src/BubbleSort.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        for(int i=0;i<arr.length-1;i++)
+        {
+            // if 1st index element is greater then swap
+            int j=0;
+            while(j<arr.length-1-i)
+            {
+                if(arr[j] > arr[j+1])
+                {
+
+                    int store_swap_value = arr[j];
```
Comment (rkpatra201) [inline src/BubbleSort.java:15]: variable naming convention not correct

**Review by rkpatra201 — APPROVED** [review]

**Comment by rkpatra201** [comment]
@Jena-Biswajit please merge this PR after addressing the comments.

File: src/BubbleSort.java (hunk 1 above)
Comment (Jena-Biswajit) [reply src/BubbleSort.java:5]: resolved

File: src/BubbleSort.java (hunk 2 above)
Comment (Jena-Biswajit) [reply src/BubbleSort.java]: Re

**Review by Jena-Biswajit — COMMENTED** [review]
Resolved Issue

**Comment by rkpatra201** [comment]
```
git reset --hard HEAD~4
git push origin HEAD -f
```

@Jena-Biswajit

**Pull request closed without merging**